### DISCUSSION TITLE
Ability to update achievement display order from the client

### DISCUSSION THREAD

**Discussion by julkasalsa** [RetroAchievements/RAIntegration#771] [Ideas]
Add new field in Achievement Editor in which the developer will put the display order number. This way the achievement could be sorted while being produced, and developer won't need to access Achievement Manager page just to do it.

**Comment by Jamiras** [comment]
display order information is not currently sent to or from the client. this will require API changes.

**Comment by meleu** [comment]
yeah, I would like to have this feature in the toolkit too.

I can take care of it on the server side.

**Comment by timenoe** [comment]
I would love to be able to be able to sort achievements in the Assets List. Even if it's just Local achievements.
